docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.3'
+
+services:
+  api_service:
+    build: ./services/api_service
+    command: python3 manage.py runserver 0.0.0.0:8000
+    volumes:
+      - ./services/api_service:/usr/src/api_service/
```

Comment:
Why you attach this as volume and copy files in Dockerfile?

Reply:
Idk, used just docker official sample for django,